connman/src/technology.c

```diff
@@ -49,6 +49,7 @@ static bool global_offlinemode;
 static unsigned int global_offlinemode_override; /* Technology bitmask */
 struct connman_access_tech_policy *tech_access_policy;
 static unsigned int enable_delayed_ids[MAX_CONNMAN_SERVICE_TYPES] = { 0 };
+static bool save_changes = true;
```

Comment:
I'm not objecting to this, but just thinking would it be better to have some kind of "changing user" flag instead.